components/header/HeaderSubNav.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styles from "./Header.module.css";
+
+export type Navigation = {
+  subNavOne: string;
+  subNavTwo: string;
+};
+
+export default function SubNav({ subNavOne, subNavTwo }: Navigation) {
+  return (
+    <section className={styles.subNav}>
+      <button
+        className={styles.navOne}
+        onClick={() => console.log("left button: clicked")}
```

Comment:
do you need this `console.log` anymore?

Reply:
I have left it in as a placeholder for the later linkings.